scripts/cola_baseline.sh

```diff
@@ -0,0 +1,20 @@
+
+# random-elmo
+rm *.out
+JIANT_OVERRIDES="exp_name = cola-elmo-baseline, run_name = random-elmo, allow_untrained_encoder_parameters = 1, allow_missing_task_map = 1, do_pretrain = 0" JIANT_CONF="config/cola-elmo.conf" sbatch ~/prince.sbatch
```

Comment:
I don't think you've shared the sbatch file.

Reply:
Add some explanation of what that file needs to be, at least. And mention how it differs from the existing ones.

Reply:
> I don't think you've shared the sbatch file.

No, I didn't. That file was used to set up environment, some path settings etc. Should I put that into jiant?

Reply:
Up to you, but this isn't appropriate as-is. The master copy of the repo needs to be clearly usable by anyone. So either share the sbatch file (and remove any paths that are specific to you) or remove all references to it.